groups/index.html

```diff
@@ -3,7 +3,13 @@
 title: BioSchemas Groups
 ---
 
-<h1>Bioschemas Groups</h1>
+<h1>Bioschemas Groups 
+<div class="git-edit" style="float: right;">
```

Comment:
I wonder if this should be at the bottom of the page rather than the top so that it is obvious once you've scrolled through all the groups.

Reply:
Carole would say that no one scrolls to the bottom of the page; I know this as Alan said exactly that to me yesterday.
We could have two? Top and bottom?